### PR TITLE
LPS-160258 TinyMCE and Simple editors were deprecated in 7.3 but they weren't removed in portal.properties file

### PR DESCRIPTION
**Pull request by jorgediaz-lr**
https://issues.liferay.com/browse/LPS-160258

TinyMCE and Simple editors were deprecated in 7.3 (see LPS-110733 and LPS-110734) but they weren't removed in portal.properties file

### CODE REVIEW

**Review by diegonvs — APPROVED** [review]

**Comment by jorgediaz-lr** [comment]
Hi @diegonvs 

Do you have any updates about this PR?

Can we forward it?

Thank you

**Comment by diegonvs** [comment]
Yes, forwarding. I was waiting for someone to give another approval but let's go 🚢

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**
